Doc/library/pathlib.rst

```diff
@@ -889,6 +889,14 @@ call fails (for example because the path doesn't exist).
 
    ``False`` is also returned if the path doesn't exist or is a broken symlink;
    other errors (such as permission errors) are propagated.
+   
+   
+.. method:: Path.is_junction()
+
+   Return ``True`` if the path points to a junction, and ``False`` for any other
+   type of file (as returned by :func:`os.path.isjunction`).
```

Comment:
Note here that junctions currently only exist on Windows, but the function is available on all platforms. (Preferably in a way like that which allows us to expand the definition in the future if necessary).

I think we'd rather not have the reference back to os.path here. The pathlib docs should have all the useful information the user needs.

Reply:
As weird as it sounds, i stuck with junctions because despite not really being documented you can create a 'directory junction' on windows that points to a file. This function still returns True in that case since the OS categorizes it as a junction... but i guess not really a 'directory junction'